predeployed/test/test_cc_generator.py

```diff
@@ -40,6 +40,7 @@ def test_default_admin_role(self, tmpdir):
             assert dc.functions.getRoleMember(ConfigControllerGenerator.DEFAULT_ADMIN_ROLE,
                                               0).call() == self.OWNER_ADDRESS
             assert dc.functions.hasRole(ConfigControllerGenerator.DEFAULT_ADMIN_ROLE, self.OWNER_ADDRESS).call()
+            assert dc.functions.version().call() == '1.0.1'
```

Comment:
I would suggest to change 1.0.1 to `get_distribution('config_controller_predeployed').version`
To prevent additional changes in future releases